src/System.Collections.NonGeneric/tests/Performance/System.Collections.NonGeneric.Performance.Tests.csproj

```diff
@@ -25,6 +25,7 @@
     <ProjectReference Include="$(CommonPath)\..\perf\PerfRunner\PerfRunner.csproj">
       <Project>{69e46a6f-9966-45a5-8945-2559fe337827}</Project>
       <Name>PerfRunner</Name>
+      <KeepProjectReference>true</KeepProjectReference>
```

Comment:
It tells the sync/generateprojectjson step to keep this project as a reference, instead of trying to generate a `project.json` for the *Performance.Tests.csproj files where it reference the PerfRunner as a package.